pages/c/_cluster/apps/charts.vue

```diff
@@ -292,22 +307,31 @@ export default {
     </header>
 
     <div class="left-right-split">
-      <div class="mt-10">
-        <Checkbox
-          :value="allRepos"
-          :label="t('catalog.charts.all')"
-          :class="{'pull-left': true, 'repo': true}"
-          @input="toggleAll($event)"
-        />
-        <Checkbox
-          v-for="r in repoOptions"
-          :key="r.label"
-          v-model="r.enabled"
-          :label="r.label"
-          :class="{'pull-left': true, 'repo': true, [r.color]: true}"
-          @input="toggleRepo(r, $event)"
-        />
+      <div>
+        <Select
+          :searchable="false"
+          :options="[{label: 'All', all: true, enabled:areAllEnabled() },...repoOptions]"
```

Comment:
`All` needs localising. Think also the array would be better coming from a computed property

Reply:
Done

Reply:
I don't think this has changed, did it get missed from a commit?